assets/scss/application.scss

```diff
@@ -9,7 +9,7 @@ $govuk-page-width: $moj-page-width;
 
 @import 'node_modules/@ministryofjustice/hmpps-connect-dps-components/dist/assets/footer';
 @import 'node_modules/@ministryofjustice/hmpps-connect-dps-components/dist/assets/header-bar';
-@import 'node_modules/hmpps-court-cases-release-dates-design/hmpps/assets/stylesheets/all';
 
 @import './components/header-bar';
 @import './local';
+@import './server/views/partials/miniProfile/_all.scss'
```

Comment:
the new file might go into `/assets/components/_mini-profile.scss` and imported as :
```
@import './components/mini-profile';
```
in the same format as header-bar above?

Reply:
It could, but that would go against keeping all relevant files in a similar place like we've been doing elsewhere.

Reply:
then can it be `@import './server/views/partials/miniProfile/all'`? i'm not familiar with scss syntax/naming convention

Reply:
Yes it can :) done